tools/development/helpers/test-python.sh

```
@@ -7,7 +7,7 @@ test_directory="${project_directory}/bindings/python/test"
 
 pushd "${test_directory}" > /dev/null
 
-    python${OSTK_PYTHON_VERSION} -m pytest -sv ${@} \
+    ${OSTK_VIRTUAL_ENV}/bin/python -m pytest -sv ${@} \
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)